test/TEST-01-BASIC/test.sh

```diff
@@ -53,7 +53,7 @@ Description=Testsuite service
 After=multi-user.target
 
 [Service]
-ExecStart=/bin/bash -c 'set -x; systemctl --failed --no-legend --no-pager > /failed ; echo OK > /testok; while : ;do echo "testsuite service waiting for journal to move to /var/log/journal" > /dev/console ; for i in /var/log/journal/*;do [ -d "\$i" ] && echo "\$i" && break 2; done; sleep 1; done; sleep 1; exit 0;'
+ExecStart=/bin/bash -c 'set -x; systemctl --failed --no-legend --no-pager > /failed ; echo OK > /testok;'
```

Comment:
Looks good, thanks!

One more point of `/bin/sh` is that we can then use busybox shell, in which case we can aim at a very minimal container that doesn't need to have bash + coreutils to run the experiments on.

Cheers!
Filipe